### PR TITLE
Officially support Window event targets in loadPromise.

### PR DESCRIPTION
**Pull request by cramforce**

### CODE REVIEW

File: src/event-helper.js [hunk 1]
```diff
@@ -93,39 +93,46 @@ export function listenOncePromise(element, eventType, opt_capture,
...
-    if (element.tagName === 'AUDIO' || element.tagName === 'VIDEO') {
-      unlistenLoad = listenOnce(element, 'loadstart', resolve);
+    const tagName = eleOrWindow.tagName;
+    if (tagName === 'AUDIO' || tagName === 'VIDEO') {
+      unlistenLoad = listenOnce(eleOrWindow, 'loadstart', resolve);
     } else {
-      unlistenLoad = listenOnce(element, 'load', resolve);
+      unlistenLoad = listenOnce(eleOrWindow, 'load', resolve);
+    }
+    // For eleOrWindows, unlisten on error (don't for Windows)
```
Comment (jridgewell) [inline src/event-helper.js]: "For elements, unlisten on error"

File: src/event-helper.js [hunk 2]
```diff
@@ -93,39 +93,46 @@ export function listenOncePromise(element, eventType, opt_capture,
 
 /**
  * Whether the specified element has been loaded already.
- * @param {!Element} element
+ * @param {!Element|!Window} element
```
Comment (dvoytenko) [inline src/event-helper.js]: Let's rename the arg as well here.

**Comment by dvoytenko** [comment]
LGTM with one request

File: src/event-helper.js (hunk 2 above)
Comment (cramforce) [inline src/event-helper.js]: Done

File: src/event-helper.js (hunk 1 above)
Comment (cramforce) [inline src/event-helper.js]: Done

**Pull request merged**